Document/0x08b-Reference-Apps.md

```diff
@@ -0,0 +1,71 @@
+# Reference applications
+
+The applications listed below can be used as training materials. Note: only the MSTG apps and Crackmes are tested and maintained by the MSTG project.
+
+## Android
+
+### Android Crackmes
+
+A set of apps to test your Android application hacking skills - <https://github.com/OWASP/owasp-mstg/tree/master/Crackmes>
+
+### AndroGoat
+
+An open source vulnerable/insecure app using Kotlin. This app has a wide range of vulnerabilities related to certificate pinning, custom URL schemes, Android Network Security Configuration, WebViews, root detection and over 20 other vulnerabilities - <https://github.com/satishpatnayak/AndroGoat>
+
+### DVHMA
+
+A hybrid mobile app (for Android) that intentionally contains vulnerabilities - <https://github.com/logicalhacking/DVHMA>
+
+### Digitalbank
+
+A vulnerable app created in 2015, which can be used on older Android platforms - <https://github.com/CyberScions/Digitalbank>
+
+### DIVA Android
+
+An app intentionally designed to be insecure which has received updates in 2016 and contains 13 different challenges - <https://github.com/payatu/diva-android>
+
+### DodoVulnerableBank
+
+An insecure Android app from 2015 - <https://github.com/CSPF-Founder/DodoVulnerableBank>
+
+### InsecureBankv2
+
+A vulnerable Android app made for security enthusiasts and developers to learn the Android insecurities by testing a vulnerable application. It has been updated in 2018 and contains a lot of vulnerabilities - <https://github.com/dineshshetty/Android-InsecureBankv2>
+
+### MSTG Android app (Java)
+
+A vulnerable Android app with vulnerabilities similar to the test cases described in this document - <https://github.com/OWASP/MSTG-Hacking-Playground/tree/master/Android/MSTG-Android-Java-App>
+
+### MSTG Android app (Kotlin)
+
+A vulnerable Android app with vulnerabilities similar to the test cases described in this document - <https://github.com/OWASP/MSTG-Hacking-Playground/tree/master/Android/MSTG-Android-Kotlin-App>
+
+## iOS
+
+### iOS Crackmes
```

Comment:
Could you please include one entry per iOS crackme from https://github.com/OWASP/owasp-mstg/tree/master/Crackmes?

Reply:
Yea, of course! It might look something like these or you meant a different entry? Let me know if it's on the right track 🌞 

<img width="870" alt="Screen Shot 2022-07-13 at 21 57 40" src="https://user-images.githubusercontent.com/17871719/178707685-f022708e-9064-48bb-aaeb-f14d7b48cb3c.png">
<img width="849" alt="Screen Shot 2022-07-13 at 21 56 53" src="https://user-images.githubusercontent.com/17871719/178707728-2c2e7d67-5715-45ac-a31b-2fa038c5db72.png">

Reply:
It’d be even better to use markdown headers to refer to them individually. For example:

```
### iOS MSTG Crackmes

#### UnCrackable App for iOS Level 1

…
```